src/app/api/streaks/route.ts

```diff
@@ -0,0 +1,60 @@
+'use server';
+import { NextResponse } from 'next/server';
+import prisma from '@/lib/db';
+
+//----------------------------------------------------------------
+// this endpoint is used to get all the streaks for a user by id.
+//----------------------------------------------------------------
+export async function GET(req: Request) {
+  try {
+    // get the url from the fetch request
+    const url = new URL(req.url);
+
+    // get the skip parameter to increment the page number
```

Comment:
Can you clarify what the page number is for?

Reply:
sorry to have not looked before I haven't used this tool that much yet